.github/workflows/certora-prover.yml

```diff
@@ -38,10 +38,10 @@ jobs:
           python-version: '3.10'
           cache: 'pip'
       - name: Install java
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
-          java-version: '11'
-          java-package: 'jre'
+          distribution: temurin
```

Comment:
What's the reason for the prover updates?

Reply:
Actually not entirely sure where these changes came from. Originally thought it was from pulling from the `Combined-Withdrawal-Flow` branch. @ChaoticWalrus Should I revert these changes?

Reply:
> What's the reason for the prover updates?

This was an attempt to fix some CI failures, which unfortunately I'm still seeing somewhat intermittently -- they're hard to track down because everything works locally.
IDK they're showing up as diffs in this PR (seems like a GH error?) because these changes are already [merged into master](https://github.com/Layr-Labs/eigenlayer-contracts/blob/master/.github/workflows/certora-prover.yml#L40-L44)

> Should I revert these changes?

No, see above, IDK why this is even showing up in diffs

Reply:
sg just seemed off